cocotb/share/makefiles/Makefile.sim

```diff
@@ -79,7 +79,15 @@ ifeq ($(MAKECMDGOALS),help)
     help_vars := $(subst %,${newline},$(shell cocotb-config --help-vars | tr \\n %))
     $(info ${help_vars})
     # is there a cleaner way to exit here?
-    $(error "Stopping after printing help")
+    $(error Stopping after printing help)
+endif
+
+COCOTB_CONFIG_CMD := $(shell :; command -v cocotb-config)
```

Comment:
This is pretty pointless - `include $(shell cocotb-config --makefiles)/Makefile.sim` will fail before running this line.

Reply:
... as mentioned in the commit message. I've seen at least one case where a user ran into this issue.

Reply:
Mind adding a comment to that effect on the makefile, so that a future cleanup doesn't remove it again?

Reply:
Also, is the premise here that users can now provide an explicit path to cocotb-config?

Reply:
> Mind adding a comment to that effect on the makefile, so that a future cleanup doesn't remove it again?


Sure, done.

> Also, is the premise here that users can now provide an explicit path to cocotb-config?

No. I updated the commit to prevent overriding this variable, but I note that this is something we don't do for other internal variables. Generally the rule is: if it is not documented, it is internal.

Reply:
It's not very commonly used, but IIRC make allows variable names to start with an underscore which would be something for another PR.